CONTRIBUTING.md

```diff
@@ -1,29 +1,18 @@
-Note: by contributing code to the Valkey project in any form, including sending
-a pull request via GitHub, a code fragment or patch via private email or
-public discussion groups, you agree to release your code under the terms
-of the Valkey license that you can find in the COPYING file included in the Valkey
-source distribution.
```

Comment:
Without this, I think we are only saying code sent via an official PR with a DCO is something that we are granted access to contribute with the license. I am OK with this, but I wanted to highlight it here to make sure that is aligned with your expectation.

Reply:
Do we accept code without a DCO?

Someone has to sign off, right? Even when we're adding code from other projects, etc.

Reply:
Yes. To contribute you need a dco for prs. The previous wording said if you sent the project an email, you were also consenting to it being distributed over bsd.

Reply:
Right. It's probably a too major change for this PR anyway. I'll put it back (but maybe not at the top).

Reply:
We're saying "We require that every contribution to Valkey to be signed with a DCO.". That's more than the old note required.

I'm adding the note back, but I'm putting it in the end of the DCO section and edit it to this:

> If you're contributing code to the Valkey project in any other form, including
sending a code fragment or patch via private email or public discussion groups,
you need to spell out that the contribution is in accordance with the DCO.

OK?

Reply:
Yeah, that makes it feel less like a major decision.